src/routes/metrics/+page.svelte

```diff
@@ -1,20 +1,20 @@
 <script lang="ts">
 	import { Table } from 'sveltestrap';
 	import MetricImplementationBlock from '$lib/MetricImplementationBlock.svelte';
-	import { getMetricImplementation } from '$lib/orchestrator';
-	import type { Metric, MetricImplementation } from '$lib/assessment';
+	import { getMetricImplementation} from '$lib/orchestrator';
+	import type { Metric, MetricImplementation} from '$lib/assessment';
 	import type { PageData } from './$types';
 
 	export let data: PageData;
-
+	
 	async function fetchImplementation(metric: Metric): Promise<MetricImplementation> {
```

Comment:
Ok looks like you c/p this code (see comment above). But I think its unncecessary.